slothy/targets/aarch64/aarch64_neon.py

```diff
@@ -2821,6 +2821,11 @@ def make(cls, src):
             ]
         return obj
 
+class cmge(AArch64Instruction): # pylint: disable=missing-docstring,invalid-name
```

Comment:
It might be worth introducing a parent class for compare instructions, like
```
class ASimdCompare(AArch64Instruction):
   """Parent class for ASIMD compare instructions"""

class cmge(ASimdCompare):
   ...
```

and then wherever the SWOG talks about all compare instructions, you can just define latency/throughput/units in one go. See the Neoverse N1 model for examples -- it makes the uArch model more readable.